project/plugins.sbt

```diff
@@ -1,3 +1,6 @@
+addSbtPlugin("org.lyranthe.sbt" % "partial-unification" % "1.1.0")
+addSbtPlugin("com.eed3si9n" % "sbt-buildinfo" % "0.9.0")
+addSbtPlugin("com.twilio" % "sbt-guardrail" % "0.46.0")
 addSbtPlugin("com.typesafe.sbt" % "sbt-native-packager" % "1.3.20")
 //addSbtPlugin("io.github.davidgregory084" % "sbt-tpolecat" % "0.1.3")
```

Comment:
Do we actually use this? If not remove.

Reply:
Actually it would be a good idea to use it, since in implies "recommended" compilation flags: https://tpolecat.github.io/2017/04/25/scalac-flags.html
But it doesn't work with the generated code since e.g. it has a lot of unused import and it won't compile then.. So, I'll remove it.